packages/components/src/components/bal-navigation/bal-meta-mobile-foot/bal-meta-mobile-foot.tsx

```diff
@@ -0,0 +1,17 @@
+import { Component, h, ComponentInterface, Host } from '@stencil/core'
+import { BEM } from '../../../utils/bem'
+
+@Component({
+  tag: 'bal-meta-mobile-foot',
+})
+export class MetaMobileFoot implements ComponentInterface {
+  render() {
+    const footEl = BEM.block('nav').element('footmobile')
+
+    return (
+      <Host class={{ ...footEl.class(), 'is-hidden-desktop': true }}>
```

Comment:
This look like a duplicate of the bal-main-nav. Could we solve that without components ?

Reply:
I am just a bit worried with the payload when we add so many small components

Reply:
unnecessary component deleted